tests/test_oc4ids_tranforms.py

```diff
@@ -0,0 +1,1332 @@
+        releases, "1", transforms=[transforms.contracting_process_setup, transforms.suppliers],
+    )
+
+    assert output["parties"] == releases[0]["parties"]
+
+    assert output["contractingProcesses"][0]["summary"]["suppliers"] == [
+        {"id": "a", "name": "A"},
+        {"id": "b", "name": "B"},
+    ]
+
+
+def test_contract_value():
+
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "awards": [
+                {"value": {"amount": 10, "currency": "USD"}},
+                {"value": {"amount": 10, "currency": "USD"}},
+                {"value": {"amount": 10, "currency": "USD"}},
+            ],
+        }
+    ]
+
+    output = transforms._run_transforms(
+        releases, "1", transforms=[transforms.contracting_process_setup, transforms.contract_price],
+    )
+
+    assert output["contractingProcesses"][0]["summary"]["contractValue"] == {"amount": 30, "currency": "USD"}
+
+    # change an currency
+    releases[0]["awards"][1]["value"]["currency"] = "CAD"
+
+    output = transforms._run_transforms(
+        releases, "1", transforms=[transforms.contracting_process_setup, transforms.contract_price],
+    )
+
+    assert "contractValue" not in output["contractingProcesses"][0]["summary"]
+
+
+def test_contracting_process_description():
+
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "contracts": [{"description": "a", "items": [{"description": "item_a"}]}],
+            "awards": [{"description": "b", "items": [{"description": "item_b"}]}],
+            "tender": {"description": "c", "items": [{"description": "item_c"}]},
+        }
+    ]
+
+    output = transforms._run_transforms(
+        releases, "1", transforms=[transforms.contracting_process_setup, transforms.contract_process_description],
+    )
+
+    assert output["contractingProcesses"][0]["summary"]["description"] == "a"
+
+    # with second contract we do not use contract description but item description
+    releases[0]["contracts"].append({"description": "a"})
+
+    output = transforms._run_transforms(
+        releases, "1", transforms=[transforms.contracting_process_setup, transforms.contract_process_description],
+    )
+
+    assert output["contractingProcesses"][0]["summary"]["description"] == "item_a"
+
+    # with second contract item we do not use contract item description but award description
+    releases[0]["contracts"][0]["items"].append({"description": "item_b"})
+
+    output = transforms._run_transforms(
+        releases, "1", transforms=[transforms.contracting_process_setup, transforms.contract_process_description],
+    )
+
+    assert output["contractingProcesses"][0]["summary"]["description"] == "b"
+
+    # with second award we do not use award description but award item description
```

Comment:
From the OC4IDS mapping:

> for contracting processes with multiple contracts or awards, copy from tender.description or tender.items.description.

Reply:
logic has changed but may require further discussion